README.md

```diff
@@ -1 +1,3 @@
 # DTS' Maximo & Geo Integration
+
+Currently, this is handled by an ETL which runs in Airflow. For more details, see the [ETL README](etl/parse_email_save_attachment/README.md).
```

Comment:
H/T to @mddilley for requesting some linkage from the top level readme down to the one particular to this ETL. 🙏